src/rgw/setup/rgw.py

```diff
@@ -1101,7 +1117,7 @@ def _update_resource_limit_based_config(conf: MappedConf, client_section: str):
            raise SetupError(errno.EINVAL,
                             'Invalid value is generated for {const.SVC_THREAD_POOL_SIZE_KEY} key.')
 
-        if concurrent_max_requests_val > 0 :
+        if int(concurrent_max_requests_val) > 0 :
```

Comment:
extra precaution ? why we are adding this now ?

Reply:
its beacause of Gconf values.
If user provide value in solution file then this type casting is required.

Reply:
ok